### PR TITLE
Create ajaypatil.md

### PR DESCRIPTION
**Pull request by patilaja**
adding ajaypatil.md to contributors folder

### CODE REVIEW

**Review by stanly-johnson — CHANGES_REQUESTED** [review]
@patilaja Can you pls refactor to the expected format, you can refer this [PR](https://github.com/Leapcode-Open/landingpage/pull/526/files)

**Review by patilaja — COMMENTED** [review]
Changed file as suggested.

File: src/contributors/ajaypatil.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+username: patilaja
```
Comment (stanly-johnson) [inline src/contributors/ajaypatil.md:1]:
```suggestion
---
username: patilaja
```

File: src/contributors/ajaypatil.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+name: Ajay Patil
+bio: Program Director. Practicing Data Science area specifically Machine Learning.
```
Comment (stanly-johnson) [inline src/contributors/ajaypatil.md:3]:
```suggestion
bio: Program Director. Practicing Data Science area specifically Machine Learning.
---
```

**Review by stanly-johnson — CHANGES_REQUESTED** [review]

**Pull request merged**